src/main/java/com/epam/esm/dao/GiftCertificateTagDao.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.esm.dao;
+
+import com.epam.esm.sql.SqlQueries;
+import com.epam.esm.entity.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.Set;
+
+@Component
+public class GiftCertificateTagDao {
+
+    private final JdbcTemplate template;
+
+    @Autowired
+    public GiftCertificateTagDao(JdbcTemplate template) {
+        this.template = template;
+    }
+
+    public void addTagId(long id, Long tagId) {
+        template.update(SqlQueries.CONNECT_TAG_TO_CERTIFICATE, id, tagId);
+    }
+
+    public void deleteTagId(long id, Long tagId) {
+        template.update(SqlQueries.DISCONNECT_TAG_TO_CERTIFICATE, id, tagId);
+    }
+
+    public void createConnections(long giftCertificateId, Set<Tag> tags) {
```

Comment:
why did you decide to have void here?

Reply:
There is no need to return something. We just make tied certificates and relevant tags.

Reply:
is here only one result action possible?

Reply:
@a1exanddrovich, please don't leave comments unanswered! 
Should be fixed, but since aren't going to have this implementation in the next module, I'd say that we can leave it as is for now.

Reply:
I assume we can get either a successful result or an unsuccessful one while tying given certificates to their tags